ui-components/src/headers/StepHeader.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react"
+import { t } from "../.."
+import "./StepHeader.scss"
+
+export interface StepHeaderProps {
+  currentStep: number
+  totalSteps: number
+  stepTitle?: string
+}
+
+const StepHeader = ({
+  currentStep,
+  totalSteps,
+  stepTitle = t("stepHeader.defaultLabel"),
+}: StepHeaderProps) => {
+  return (
+    <div className="step-header">
+      <div className="circle-number">{currentStep}</div>
+      <div>{`out of ${totalSteps}`}</div>
```

Comment:
We'll need to pass this string in as a prop, so that we don't have any untranslated strings